modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -0,0 +1,117 @@
+/**
```

Comment:
I think if you rename files do that before you do other changes in the same commit and it should be easier to diff. If more than 70% of the file is changed it not picked up as a rename. But I think if you do it in two steps it is.

Reply:
Good to know, I wasn't aware of that so I'll try and remember to give it a go in future.